cli/cli.go

```diff
@@ -400,6 +401,10 @@ func (ec *ExecutionContext) Validate() error {
 		return errors.Wrap(err, "validating current directory failed")
 	}
 
+	// load .env file
+	envfile := filepath.Join(ec.ExecutionDirectory, ".env")
+	_ = godotenv.Load(envfile)
```

Comment:
@ShraddhaAg can we capture the error and log it in debug level if the file is found?

Reply:
If user defined name of .env file is given, error is returned else logged if the file is found.